### ISSUE TITLE
Annotations no longer hide before load

### ISSUE THREAD

**Issue by gjcope** [google/model-viewer#1419]
### Description

With the newest release, annotations seem to no longer hide before load using the style code from https://modelviewer.dev/examples/annotations.html 

`/* This keeps child nodes hidden while the element loads */
  :not(:defined) > * {
    display: none;
  }`

#### Live Demo

https://canyon-tourmaline-titanoceratops.glitch.me/

### Browser Affected

- [x] Chrome 84.0.4147.105
- [ ] Edge
- [x] Firefox  78.0.2
- [ ] Helios
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [x] Windows 10

### Versions

- model-viewer: v1.1.0
- three.js: r0.117.1

**Comment by elalish** [comment]
@gjcope Actually, that line only keeps them hidden until the model-viewer element itself is loaded, which is quick. Then it is the poster which hides the annotations while the 3D model is loading. The reason they're visible is because you set your poster-color to transparent. I would recommend using https://modelviewer.dev/editor/ to create a proper poster image and set poster-color to match your background color.

**Comment by gjcope** [comment]
Got it, thanks. We actually were using the old interactive page to create the poster (what makes the new one 'proper'?) and using transparent as indicated in the --poster-color documentation since we have a gradient background and wanted it to remain seamless. Any recommendations other than to not use a gradient?

**Comment by elalish** [comment]
Oh, I must have just not noticed. The new page makes just the same kind of posters as the interactive page did, so no worries there. Gradient background with annotations is a little tricky; I'll have to think if there's a way we could make that easier.

**Comment by gjcope** [comment]
@elalish Your explanation makes sense, but I still think there is something going on. If you take a look at the glitch, I've added a poster with a transparent background to the first model viewer instance and kept the transparent style. I changed the unpkg model-viewer version to explicitly load 1.0.0 and the annotations are hidden until the model loads. If I change it to 1.1.0 the annotations are now visible during the poster view. The effect happens when only changing the model-viewer version...
